Source/DafnyStandardLibraries/src/DafnyStdLibs/Collections/Seqs.dfy

```diff
@@ -949,4 +916,11 @@ module DafnyStdLibs.Collections.Seq {
 
       [right[0]] + MergeSortedWith(left, right[1..], lessThanOrEq)
   }
+
+  lemma LemmaNewFirstElementStillSortedBy<T>(newFirst: T, s: seq<T>, lessOrEqual: (T, T) -> bool)
```

Comment:
Moved this here since it's really only useful as a helper lemma for MergeSortedWith and not terribly general.